packages/core/core/src/utils/template_generator.js

```diff
@@ -306,4 +311,9 @@ class TemplateGenerator {
     });
   }
 }
-module.exports = TemplateGenerator;
+
+function extractZip(filename, packageDirectory, opts, cb) {
+  decompress(filename, packageDirectory, opts).then((_files) => {
+    cb();
+  });
+}
```

Comment:
This function used to live in `lib/utils/utils` but was nowhere else used, so I inlined it here.